brianlong/www.validators.app
# 184246443 update sidekiq to v7

#### What's this PR do?
- as in title

#### How should this be manually tested?
- run bundler
- run tests
- run sidekiq and check if it performs jobs without errors

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184246443)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [Y] Does this PR add new dependencies?
